app/controllers/spree/admin/states_controller_decorator.rb

```diff
@@ -0,0 +1,12 @@
+Spree::Admin::StatesController.class_eval do
+
+  def index
+    respond_with(@collection) do |format|
+      format.html
+      format.js { render :partial => 'state_list' }
+
+    Spree::Config.set(params[:state_display])
```

Comment:
Hopefully this can be managed integrating rubocop rules in CI. Really looking forward to start that discussion.